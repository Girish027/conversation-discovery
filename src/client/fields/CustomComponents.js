import CustomTextArea from './CustomTextArea';
import CustomTextField from './CustomTextField';
import CustomUpDown from './CustomUpDown';

export default {
  text: CustomTextField,
  textarea: CustomTextArea,
  updown: CustomUpDown,
  noop: () => null
};
