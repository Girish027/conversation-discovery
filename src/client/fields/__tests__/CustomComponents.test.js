import CustomComponents from '../CustomComponents';

describe('CustomComponents', function () { 
  test('renders correctly for default props', () => {
    expect(CustomComponents).toMatchSnapshot();
  });
});