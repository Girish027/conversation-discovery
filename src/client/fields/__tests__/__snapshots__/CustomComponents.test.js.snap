// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomComponents renders correctly for default props 1`] = `
Object {
  "noop": [Function],
  "text": [Function],
  "textarea": [Function],
  "updown": [Function],
}
`;
